pkg/volume/gitrepo/git_repo_test.go

```diff
@@ -113,14 +113,14 @@ func TestPlugin(t *testing.T) {
 				Name: "vol1",
 				VolumeSource: v1.VolumeSource{
 					GitRepo: &v1.GitRepoVolumeSource{
-						Repository: gitUrl,
+						Repository: gitURl,
```

Comment:
```suggestion
						Repository: gitURL,
```
I think we missed capitalizing this final `L`.